### PR TITLE
Change clickable area of search results

### LINKED ISSUE #34: Only Title on Search Result should be clickable
Currently, when we search for documents, the entire box of the result is clickable and leads to the document. We should change this so that only the title is clickable.

### PR DESCRIPTION
**Pull request by m-doughty**
### Link to Github issue

resolves #34 

### Acceptance Criteria

- [x] Only the title of search results should take you to the document.

### Testing information

1. Navigate to the website.
2. Perform a valid search which returns results (`ibuprofen` for example).
3. Clicking on the title (or file size) should load the document, clicking anywhere else should not.

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/web/components/search-results/index.tsx [hunk 1]
```diff
@@ -165,26 +165,26 @@ const SearchResults = (props: { drugs: IDocument[]; lastSearch: string }) => (
               <p className="icon">{drug.docType.toUpperCase()}</p>
             </div>
             <div className="right">
-              <a href={drug.url}>
-                <p className="drug-name">
+              <p className="drug-name">
```
Comment (pataruco) [inline medicines/web/components/search-results/index.tsx]: Please, if this a title of a result item I suggest to change `<p>` for a heading tag, e.g `<h3>`

**Comment by pataruco** [comment]
For Skateboard lite is OK, but I think a more semantically accurate tags for this would be [`<dl>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

File: medicines/web/components/search-results/index.tsx (hunk 1 above)
Comment (pataruco) [reply medicines/web/components/search-results/index.tsx]: Resolved

Comment (m-doughty) [reply medicines/web/components/search-results/index.tsx]: Agreed, we should raise an issue to review semantic use of HTML but this is out of scope of this bugfix.

**Comment by m-doughty** [comment]
@pataruco I've raised issue #39 so that we can ensure we're using correct semantic HTML going forward.

**Pull request merged**